python-backend/app/api/incidents/models/mine_incident.py

```diff
@@ -70,11 +70,16 @@ class MineIncident(AuditMixin, Base):
         db.ForeignKey(
             'mine_incident_followup_investigation_type.mine_incident_followup_investigation_type_code'
         ))
+    mine_determination_type_code = db.Column(
```

Comment:
Should this be `incident_determination_type_code` rather than `mine_determination_type_code`?

Reply:
It's being used to indicate the determination that a representative at the mine made (in contrast to an inspector -- which is held in the column `determination_type_code`), so I felt that this name reflected that.

Reply:
I see what you mean. Its almost like we need the possessive "mine's" in there. But I'm fine with this.